nativelib/src/main/scala/scala/scalanative/native/alwaysinline.scala

```diff
@@ -0,0 +1,5 @@
+package scala.scalanative
+package native
+
+/** Unconditionally inline annotated method. */
+final class alwaysinline extends scala.annotation.StaticAnnotation
```

Comment:
If it was only SJS, I wouldn't necessarily recommend it. But since the JVM also treats it like that, it has sort of acquire a portable meaning already. It would be good to preserve that portability across SN as well.

You could add `@inlinehint` to mean the LLVM `inlinehint`.